github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -291,7 +291,7 @@ presubmits:
                   - "true"
               topologyKey: kubernetes.io/hostname
       containers:
-      - image: kubevirtci/bootstrap:v20201119-a5880e0
+      - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
```

Comment:
This change doesnt work well
it create a different cluster instead of kind sriov, and runs all the tests

Reply:
`/bin/bash: apt: command not found` this is the problem i guess

Reply:
created PR to catch those easier and faster
https://github.com/kubevirt/kubevirt/pull/5266

Reply:
Removed this job from this PR and created a dedicate PR to fix this lane